lib/nexus_cli/mixins/task_actions.rb

```diff
@@ -0,0 +1,247 @@
+require 'json'
+require 'date'
+
+module NexusCli
+  # @author Leo Simons <lsimons@schubergphilis.com>
+  module TaskActions
+
+    # Gets information about the current Nexus scheduled tasks.
+    #
+    # @param  header a header to send with the request (Accept: application/xml by default)
+    # @return [String] a String of XML with data about Nexus tasks
+    def get_tasks(header=XML_ACCEPT_HEADER)
+      response = nexus.get(nexus_url("service/local/schedules"), :header => header)
+      case response.status
+        when 200
+          return response.content
+        else
+          raise UnexpectedStatusCodeException.new(response.status)
+      end
+    end
+
+    def create_download_indexes_task(name, schedule='daily', enabled=true, start_date=nil, recurring_time=nil,
+                                     repository_id='all_repo')
+      params = common_task_params('DownloadIndexesTask', enabled, name, recurring_time, schedule, start_date,
```

Comment:
I wonder if it might help organize this code a bit if all of the "task types" were pulled up at setup as constants at the top.

ex: `DOWNLOAD_INDEX_TASK = 'DownloadIndexesTask'`